src/sentry/eventtypes/base.py

```diff
@@ -54,4 +58,13 @@ def extract_metadata(self, data):
         else:
             title = "<unlabeled event>"
 
-        return {"title": title}
+        return {"message_title": title}
```

Comment:
@untitaker I'm not very confident about this change - Do parts of the code depend on "title" being present in metadata?

Reply:
no, I think this is fine. I would also pull up `metadata.get("title")` into `DefaultEvent.compute_title` from `BaseEvent`, to be perfectly honest. I don't see how `metadata.title` can exist for any other type than default.

Reply:
> I don't see how metadata.title can exist for any other type than default.

What about this `custom-event-title` feature?

https://github.com/getsentry/sentry/blob/ff529e299c1751c72a67b606fc04b4340d9f55f4/static/app/utils/events.tsx#L84-L86

Reply:
right so it *can* be there if somebody sets a custom title.. in those cases it is still honored because of the `get_metadata` logic. It's just that now you also write a `message_title` attribute in addition, but `title` takes precedence in `get_title`